typescript/vscode-ext/packages/vscode/src/panels/WebPanelView.ts

```diff
@@ -201,8 +201,9 @@ export class WebPanelView {
           }
           case 'jumpToFile': {
             try {
+              console.log('jumpToFile', message.data)
               const span = message.data as StringSpan
-              const uri = vscode.Uri.file(span.source_file)
+              const uri = vscode.Uri.parse(span.source_file)
```

Comment:
What does the file path loom like here? Windows wont work if we use this on a non "file:///" path.

I made this note for ourselves; https://www.notion.so/gloochat/windows-uri-treatment-fe87b22abebb4089945eb8cd1ad050ef?pvs=4

Reply:
it is a file:/// fyi!